guides/keras-tuner/distributed_tuning.py

```diff
@@ -0,0 +1,137 @@
+"""
+Title: Distributed Tuning on Multiple GPUs and Machines
+Author: Tom O'Malley, Haifeng Jin
+Date created: 2019/10/24
+Last modified: 2021/06/02
+Description: Tuning the hyperparameters of the models with multiple GPUs and multiple machines.
+"""
+"""
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-threaded CPU instance (or alternatively as a separate process on one of the workers).
```

Comment:
Please shorten the lines in the `py` file to be ~80 chars or so, it makes review and source control easier. No need to regenerate the other files, since we don't use them for reviewing or source control